docker-compose.yml

```
@@ -6,10 +6,18 @@ services:
       - '8086:8086'
     volumes:
       - influxdb-storage:/var/lib/influxdb

```

```suggestion
      - influxdb-storage:/var/lib/influxdb2
```

https://hub.docker.com/_/influxdb